src/Features/Onboarding.php

```diff
@@ -1108,6 +1114,30 @@ public static function reset_task_list() {
 		exit;
 	}
 
+	/**
+	 * Reset the extended task list and redirect to the dashboard.
+	 */
+	public static function reset_extended_task_list() {
```

Comment:
I wasn't able to add tests for these methods easily. I think that it's related to the `wp_safe_redirect` [at the end of the method](https://github.com/woocommerce/woocommerce-admin/pull/6081/files#diff-512e7bd9a37734b0a1e95b579a48226e1628f8ebf6b3a5ed7a7c80c8d0c4e55fR1137).
Since this method is very similar to the method [`reset_task_list`](https://github.com/woocommerce/woocommerce-admin/blob/main/src/Features/Onboarding.php#L1089), what do you think about adding these tests in a follow-up PR?